addons/io_hubs_addon/components/operators.py

```diff
@@ -670,6 +671,273 @@ def invoke(self, context, event):
         return {'RUNNING_MODAL'}
 
 
+class BakeLightmaps(Operator):
+    bl_idname = "object.bake_lightmaps"
+    bl_label = "Bake Lightmaps"
+    bl_description = "Bake lightmaps of selected objects using the Cycles render engine and pack them into the .blend."
+    bl_options = {'REGISTER', 'UNDO'}
+
+    # Global vars for modal operator
+    _timer = None
+    done = False
+    bake_started = False
+    cycles_samples_tmp = 1024
+    render_engine_tmp = 'CYCLES'
+    render_file_format_tmp = 'JPG'
+    bake_settings_before = False
+    lightmap_texture_nodes = []
+
+    image_type: EnumProperty(items=(('0', 'HDR', ''), ('1', 'JPEG', '')),
```

Comment:
If you use the names as the identifiers, you can access them directly, e.g. `self.image_name` will return either `HDR` or `JPEG`.
```suggestion
    image_type: EnumProperty(items=(('HDR', 'HDR', ''), ('JPEG', 'JPEG', '')),
```